### ISSUE TITLE
Error message found: "StateError: TypeError: 'undefined' is not a function (evaluating 'value.match(regex)')"

### ISSUE THREAD

**Issue by chaochanas** [bem/gemini#105]
Hi all,

I'm new to Gemini, start using it last week. But It fails to gather screen from the site I'd like to test  [AsiaWebDirect](http://www.asiawebdirect.com) with error message "StateError: TypeError: 'undefined' is not a function (evaluating 'value.match(regex)')". I've tried other site such as google.com and yandex.com as in example, it works perfectly. 

Thank you in advance for any help. If you need any further information, please feel free to let me know.

**Shell output error message**
```
...
 > CALL eval("__gemini.prepareScreenshot([\"#web-body > div.global_pink-header\"], {\"coverage\":false})") 
 > POST /session/:sessionID/execute {"script":"return __gemini.prepareScreenshot([\"#web-body > div.global_pink-header\"], {\"coverage\":false});","args":[]}
 > RESPONSE eval("__gemini.prepareScreenshot([\"#web-body > div.global_pink-header\"], {\"coverage\":false})") {"error":"JS","message":"TypeError: 'undefined' is not a function (evaluating 'value.match(regex)')\n    at parseBoxShadow (:136)\n    at getElementRect (:118)\n    at getScreenshotRect (:73)\n    at prepareScreenshotUnsafe (:19)\n    at prepareScreenshot (:9)\n    at anonymous (:1)\n    at La (phantomjs://webpage.evaluate():13)\n    at phantomjs://webpage.evaluate():13\n    at phantomjs://webpage.evaluate():13\n    at phantomjs://webpage.evaluate():14\n    at phantomjs://webpage.evaluate():14"}
✘ awd-search state_one [phantomjs]
StateError: TypeError: 'undefined' is not a function (evaluating 'value.match(regex)')
    at parseBoxShadow (:136)
    at getElementRect (:118)
    at getScreenshotRect (:73)
    at prepareScreenshotUnsafe (:19)
    at prepareScreenshot (:9)
    at anonymous (:1)
    at La (phantomjs://webpage.evaluate():13)
    at phantomjs://webpage.evaluate():13
    at phantomjs://webpage.evaluate():13
    at phantomjs://webpage.evaluate():14
    at phantomjs://webpage.evaluate():14
    at /Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/lib/browser/index.js:165:37
    at _fulfilled (/Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/node_modules/q/q.js:787:54)
    at self.promiseDispatch.done (/Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/node_modules/q/q.js:816:30)
    at Promise.promise.promiseDispatch (/Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/node_modules/q/q.js:749:13)
    at /Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/node_modules/q/q.js:557:44
    at flush (/Volumes/Macintosh HD 2/TestAutomation/Gemini/node_modules/gemini/node_modules/q/q.js:108:17)
    at process._tickCallback (node.js:419:13)
 > CALL quit() 
 > DELETE /session/:sessionID 
```

**Software version:**
\# gemini --version
0.9.5
\# phantomjs --version
1.9.8

**Configuration files:**
*.gemini.yml*
```javascript
rootUrl: http://www.asiawebdirect.com/
gridUrl: http://localhost:4444/wd/hub
browsers:
  phantomjs: phantomjs
```

*awd.suite*
```javascript
var gemini = require('gemini');

gemini.suite('awd-search', function(suite) {
    suite.setUrl('/')
        .setCaptureElements('#web-body > div.global_pink-header')
        .capture('state_one');
});
```

**Comment by SevInf** [comment]
Hi. Thanks for the report!
It was a bug in `gemini`, it is fixed in 0.9.6.
Please, upgrade and let me know if it helps.

**Comment by chaochanas** [comment]
Hi,

I've updated and run the test right - it works as expected. So I can continue explore the Gemini.

Thank you very much.
